web/db/actions/Post.ts

```diff
@@ -82,6 +82,26 @@ async function getResizedUploadUrl(uuid: string): Promise<string> {
   return `${consts.baseUrl}/api/resizedUpload/${token}`;
 }
 
+async function deleteAttachments(keysToDelete: string[]) {
+  //will make key splitting more efficient later
+  const objectsToDelete = keysToDelete.map((keyToDelete) => ({
+    Key:
+      keyToDelete.split("/")[keyToDelete.split("/").length - 2] +
+      "/" +
+      keyToDelete.split("/")[keyToDelete.split("/").length - 1],
+  }));
+  const deleteObjectsCommand = new DeleteObjectsCommand({
+    Bucket: consts.storageBucket,
+    Delete: { Objects: objectsToDelete },
+  });
+  try {
+    const returned = await storageClient.send(deleteObjectsCommand);
+    return { success: true };
+  } catch (error) {
+    return { success: false };
+  }
```

Comment:
I think it'd be useful to have some retry logic here eventually too - `DeleteObjectsCommand` has a `quiet` mode that you can use to only get the keys for which the deletion request failed. If there are any keys in there, we should keep retrying (up to 3 times)

Reply:
Okay, will keep that in mind when I implement the rest of the function. Thanks!

Reply:
So I have a couple of questions related to this. If we have a couple of images that we delete associated with a post, but not all of them- how do we proceed? Do we continue with deleting the object or do we try to restore the pictures we deleted because we couldn't delete all of them?

Reply:
We need to honor the customer's intent to delete the post, so we won't need to restore them - you can go ahead with deleting the post document. Ideally, we would mark the images that failed to delete in some way so a cron job can delete them later, but I'm not sure if we'll have the time to do that this semester.

I think for this ticket, as long you have the retry logic, we'll be good.